packages/react-native-dogfood/src/screens/Meeting/MeetingScreen.tsx

```diff
@@ -1,6 +1,10 @@
 import React, { useEffect } from 'react';
 import { NativeStackScreenProps } from '@react-navigation/native-stack';
-import { ActiveCall, ActiveCallProps } from '@stream-io/video-react-native-sdk';
+import {
+  ActiveCall,
+  ActiveCallProps,
+  useActiveCall,
```

Comment:
`useActiveCall` was deprecated. Please pull `main`, and use `useCall` if applicable to your use case.